### DISCUSSION TITLE
Autocompletion can't be disabled?

### DISCUSSION THREAD

**Discussion by cytdciytcYfouygiujoim** [dbeaver/dbeaver#17487] [Q&A]
I'd like to code without the keyword END being forcibly inserted whenever I type BEGIN or CASE.
I've disabled all of the distracting autocompletion options I can find but this remains.
Any ideas?

**Comment by LonwoLonwo** [comment]
As I can see, BEGIN/CASE adding cannot be disabled with settings.

You can create a ticket about it.
